tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -471,8 +471,10 @@ bool FindContractionWithBiasAndActivation(
   // Currently, only matmul + bias + tanh is enable
   if (!IsMatMul(*contraction_node_def) && IsTanh(*node_def)) return false;
 
-  // Currently, only conv + bias + leakyrelu is enabled
-  if (!IsConv2D(*contraction_node_def) && IsLeakyRelu(*node_def)) return false;
+  // Currently, only (conv | matmul) + bias + leakyrelu is enabled
+  if ((!IsConv2D(*contraction_node_def) && !IsMatMul(*contraction_node_def)) &&
```

Comment:
Nit: This is slightly easier to read and is more consistent with the comment.
```suggestion
  if (!(IsConv2D(*contraction_node_def) || IsMatMul(*contraction_node_def)) &&
```

Reply:
Thanks for the advice.  Code is changed.